src/hooks/useLocalTodoList.js

```diff
@@ -0,0 +1,17 @@
+import { useState, useEffect } from 'react';
+
+function useLocalTodoList(initialData) {
```

Comment:
If you wanted to make this function more generalizable, you could pass in two parameters - data and key. So you'd pass in the key as `todoList` and it would retrieve/store your data in local storage at that key. Then you could use this for an items you wanted to store in local storage.